src/Base/XMLTools.h

```diff
@@ -210,6 +216,11 @@ inline XUTF8Str::XUTF8Str(const char* const fromTranscode)
 
 inline XUTF8Str::~XUTF8Str() = default;
 
+#define XUTF8StrLiteral(literal)                                                                   \

```

Comment:
```suggestion
// Uses the compiler to create a cache of transcoded string literals so that each subsequent call
// can re-use the data from the lamda's initial creation. Permits the same usage as 
// XStr("literal").unicodeForm()
#define XUTF8StrLiteral(literal)                                                                   \
```